patches/net/minecraft/data/models/ItemModelGenerators.java.patch

```diff
@@ -9,22 +9,24 @@
      public static final ResourceLocation TRIM_TYPE_PREDICATE_ID = ResourceLocation.withDefaultNamespace("trim_type");
      public static final List<ItemModelGenerators.TrimModelData> GENERATED_TRIM_MODELS = List.of(
          new ItemModelGenerators.TrimModelData("quartz", 0.1F, Map.of()),
-@@ -39,23 +_,31 @@
+@@ -39,23 +_,33 @@
          new ItemModelGenerators.TrimModelData("amethyst", 1.0F, Map.of())
      );
      public final BiConsumer<ResourceLocation, Supplier<JsonElement>> output;
 +    @org.jetbrains.annotations.Nullable public final net.neoforged.neoforge.common.data.ExistingFileHelper fileHelper;
++    public final String modId;
  
 -    public ItemModelGenerators(BiConsumer<ResourceLocation, Supplier<JsonElement>> p_125082_) {
```

Comment:
Switch around the order of the constructors to make the patch a bit more readable